src/bsoncxx/view_or_value.hpp

```diff
@@ -118,7 +118,7 @@ class view_or_value {
         return _view;
     }
 
-   private:
+   protected:
```

Comment:
Done, I've opened [CXX-801](https://jira.mongodb.org/browse/CXX-801) to track that change.